_posts/2021-10-06-what-is-seraphis.md

```diff
@@ -0,0 +1,57 @@
+
+Privacy-focused transaction protocols have two core structural rules.
+
+1. How are amounts represented?
+2. How are key images (a.k.a. 'linking tags') constructed?
+
+Ever since RingCT was introduced, transaction protocols have had 'hidden amounts' by adopting the Confidential Transactions technique (the CT in RingCT). Key images, however, have seen recent innovations.
+
+[Triptych](https://eprint.iacr.org/2020/018) is a fourth-generation privacy-focused transaction protocol (following the Bitcoin, CryptoNote, and then RingCT models) with a new key image construction that permits 'one-of-many proofs' (which behave the same as ring signatures) with significantly better performance for large reference sets (large numbers of decoys) compared to what is possible with CryptoNote/RingCT-style key images. Note that, while the Triptych paper doesn't make a distinction between a 'transaction protocol' and a 'transaction protocol abstraction', Triptych represents a new abstract model following after RingCT.
+
+Seraphis is, similarly, a fourth-generation privacy-focused transaction protocol (abstraction). Also like Triptych, it defines a new key image construction that permits efficient one-of-many proofs. However, there are a number of notable differences between Triptych and Seraphis.
+
+
+### Triptych vs. Seraphis
+
+- Triptych
+    - Pros
+        - If a cryptocurrency uses RingCT, it is less work to upgrade to Triptych than Seraphis.
+        - Compatible with CryptoNote user addresses.
+    - Cons
+        - Multisig is [much harder](https://github.com/cypherstack/triptych-multisig) with Triptych-style key images than with either CryptoNote or Seraphis key images.
+- Seraphis: when thinking about a possible implementation...
+    - Pros
+        - Multisig can be simple, comparable to multisig with MLSAG/CLSAG.
+        - 'Membership proof delegation' is possible without leaking users' private keys. This potentially allows the following:
+            - Transaction chaining (required for full-featured atomic swaps).
+            - Offload membership proof construction to a third party, or a view-only wallet (possibly useful for spend-only hardware wallets where it is expensive/difficult to implement complex algorithms).
+            - Ignore 10-block lock time when transacting with a **trusted party** (i.e. allow them to make your tx's membership proofs and submit the tx to the network on your behalf).
+        - Flexible multi-tier address schemes. The following features can be designed into a Seraphis user address scheme (there are at least 7 design variations, half of which require 3-key addresses [a 50% increase in address length over current addresses]):
+            - View-only wallet that can see spent outputs.
+                - Especially helpful for multisig and hardware wallets, since in RingCT and CryptoNote it is necessary to constantly 'export key images'.
```

Comment:
Not needed nested bullet point.
```suggestion
        - View-only wallet that can see spent outputs. Especially helpful for multisig and hardware wallets, since in RingCT and CryptoNote it is necessary to constantly 'export key images'.
```